test/powershell/Modules/Microsoft.PowerShell.Security/CertificateProvider.Tests.ps1

```diff
@@ -116,6 +123,9 @@ Describe "Certificate Provider tests" -Tags "Feature" {
         }
         It "Should be able to get DnsNameList of certifate by path: <path>" -TestCases $currentUserMyLocations {
             param([string] $path)
+
+            if ($SetupFailure) { Set-ItResult -Inconclusive -Because "Test certificates are not installed"}
```

Comment:
```suggestion
            if ($SetupFailure) {
                Set-ItResult -Inconclusive -Because "Test certificates are not installed"
            }
```